etc/kayobe/stackhpc-overcloud-host-images.yml

```diff
@@ -0,0 +1,36 @@
+---
+##############################
+# Release train overcloud host image sources
+
+# Whether or not to download overcloud host images from Ark
+stackhpc_download_overcloud_host_images: false
+
+# Whether or not to use images with MLNX_OFED installed (for deployment using
+# mellanox/Nvidia NICs). Only available for Ubuntu Jammy and Rocky Linux 9
+# OFED images are currently WIP and this variable is a placeholder
+stackhpc_overcloud_host_image_is_ofed: false
+
+# The overcloud host image source, defined by os_distribution, os_release,
+# stackhpc_overcloud_host_image_is_ofed, and the current stable version.
+stackhpc_overcloud_host_image_url: "{{ stackhpc_release_pulp_content_url_with_auth }}/kayobe-images/\
+                                    {{ openstack_release }}/{{ os_distribution }}/{{ os_release }}/\
+                                    {{ 'ofed/' if stackhpc_overcloud_host_image_is_ofed else '' }}\
+                                    {{ stackhpc_overcloud_host_image_version }}/\
+                                    overcloud-{{ os_distribution }}-{{ os_release }}\
+                                    {{ '-ofed' if stackhpc_overcloud_host_image_is_ofed else '' }}.qcow2"
+
+# Overcloud host image versioning tags
+# These images must be in SMS, since they are used by our AIO CI runners
+stackhpc_centos_8_stream_overcloud_host_image_version: "yoga-20230525T095243"
+stackhpc_rocky_8_overcloud_host_image_version: "yoga-20230629T135322"
+stackhpc_rocky_9_overcloud_host_image_version: "yoga-20230515T145140"
+stackhpc_ubuntu_focal_overcloud_host_image_version: "yoga-20230609T120720"
+stackhpc_ubuntu_jammy_overcloud_host_image_version: "yoga-20230609T120720"
```

Comment:
It will be easier to automate updates to these versions if they are in a separate file, like pulp-repo-versions.yml

Reply:
I've moved them across to pulp-host-image-versions.yml